mediacapture-streams/message-enumerateddevices.js

```diff
@@ -0,0 +1,7 @@
+onmessage = async e => {
+  await navigator.mediaDevices.getUserMedia({audio: true, video: true});
+  const devices = await navigator.mediaDevices.enumerateDevices();
```

Comment:
I'd like to see us hold the stream alive across the enumerateDevices call here for correctness:
```suggestion
  const stream = await navigator.mediaDevices.getUserMedia({audio: true, video: true});
  const devices = await navigator.mediaDevices.enumerateDevices();
  stream.getTracks().forEach(t => t.stop());
```